about.html

```diff
@@ -0,0 +1,178 @@
+            <a href="#">English</a>
+          </li>
+        </ul>
+      </div>
+
+      <nav>
+        <div class="nav_container">
+          <a href="index.html#home" id="brand">
+            <img class="header_logo" src="images/nutrition-logo.png" alt="Brand Logo">
+          </a>
+          <ul class="desktop desktop-nav">
+            <li class="nav-links">
+              <a href="index.html#program_section">Programs</a>
+            </li>
+            <li class="nav-links">
+              <a href="index.html#speakers_section">Speakers</a>
+            </li>
+            <li class="nav-links">
+              <a href="index.html#partners">Sponsor</a>
+            </li>
+            <li class="nav-links">
+              <a href="about.html">About</a>
+            </li>
+              <li class="nav-links campaign_link">
+                <a href="#">Campaign</a>
+              </li>
+          </ul>
+          <div class="mobile-menu-bars mobile" id="menu-bars">
+            <div class="bar1"></div>
+            <div class="bar2"></div>
+            <div class="bar3"></div>
+          </div>
+        </div>
+      </nav>
+
+      <nav class="overlay overlay-slide-left" id="overlay">
+        <ul class="mobile-nav-items">
+          <li id="nav-1"><a href="index.html#home" class="nav-mobile-links">Home</a></li>
+        <li id="nav-1"><a href="index.html#program_section" class="nav-mobile-links">Programs</a></li>
+        <li id="nav-2"><a href="index.html#speakers_section" class="nav-mobile-links">Speakers</a></li>
+        <li id="nav-3"><a href="index.html#partners" class="nav-mobile-links">Sponsor</a></li>
+        <li id="nav-3"><a href="about.html" class="nav-mobile-links">About</a></li>
+        <li id="nav-3"><a href="#" class="nav-mobile-links">Campaign</a></li>
+        </ul>
+      </nav>
+    </header>
+  
+
+    <section class="about_main_header_div">
+      
+      <div class="about_header_text_container">
+        <h1 class="about_header_text"> 
+          Together preventing diabetes 2022
+        </h1>
+      </div>
+
+      <div class="about_header_divider_line"></div>
+
+      <p class="about_header_p">
+        A joyful celebration believing in the value of openness and sharing,
+            creating a positive change with people for over 80 countries is taking place in august in Cameroon.
+        We've learned that learning can be enjoyable and interactive as a result of 
+        all of this, and we're delighted to provide the platform that educates people.
+        Are we prepared for the changes?
+      </p>
+
+      <h3 class="contact_us_text">
+        Please contacts us via email  for any further questions 
+        about diseases, heakth  and the conference 2022.
+      </h3>
+      <h3 class="email_address"><a href="#">
+        emmanuellangufor@gmail.com <span><img class="fordward_arrow" src="images/icons/icons_forward.png" alt="forward arrow"></span>
+        </a>
+      </h3>
+        
+    </section>
```

Comment:
Please note that there are some discrepancies between the about page for your site and the home page in the design template. Please try to fix this and try to replicate the design template as closely as possible, and pay keen attention to the fonts, font size, colors, margins e.t.c 🙏 🙏. Replicating exact design specs give confidence to recruiters in your ability to build exact design templates 😄 😄
Your project (Pay attention to the font size of your title header 😄):
![about student](https://user-images.githubusercontent.com/73607512/192114690-40e6f9d6-9d74-46f3-b0a9-9d104ce5eaff.PNG)
Deign Template:
![about figma](https://user-images.githubusercontent.com/73607512/192114696-ac462146-3b9b-445b-8b0e-ffb8a9ecca31.PNG)